src/wallet/rpcwallet.cpp

```diff
@@ -454,9 +457,10 @@ UniValue sendtoaddress(const JSONRPCRequest& request)
     ObserveSafeMode();
     LOCK2(cs_main, pwallet->cs_wallet);
 
-    CBitcoinAddress address(request.params[0].get_str());
-    if (!address.IsValid())
-        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid Bitcoin address");
+    CTxDestination dest = DecodeDestination(request.params[0].get_str());
+    if (!IsValidDestination(dest)) {
+        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid address");
```

Comment:
nit: Any reason why this was changed from "Invalid Bitcoin address" to "Invalid address" and the other places left as is. This is part of a error string, so shouldn't break anything I guess.

Reply:
@achow101 regarding the need for error testing.